Makefile

```diff
@@ -78,8 +78,14 @@ llvm-build: llvm-build/build.ninja
 
 # Build the Go compiler.
 build/tinygo:
-	@if [ ! -f llvm-build/bin/llvm-config ]; then echo "Fetch and build LLVM first by running:\n  make llvm-source\n  make llvm-build"; exit 1; fi
+	@if [ ! -f $(LLVM_BUILDDIR)/bin/llvm-config ]; then echo "Fetch and build LLVM first by running:\n  make llvm-source\n  make llvm-build"; exit 1; fi
+	# Generate src/runtime/internal/sys/zversion.go with TinyGo version number, for GoLand and IDEA
+	echo 'package sys' > src/runtime/internal/sys/zversion.go
+	echo >> src/runtime/internal/sys/zversion.go
+	echo "const TheVersion = \`go`grep '^const version' version.go|sed 's/const version = "//'|sed 's/"//'`\`" >> src/runtime/internal/sys/zversion.go
 	CGO_CPPFLAGS="$(CGO_CPPFLAGS)" CGO_CXXFLAGS="$(CGO_CXXFLAGS)" CGO_LDFLAGS="$(CGO_LDFLAGS)" go build -o build/tinygo -tags byollvm .
+	# GoLand and IDEA expect a "go" binary under bin
+	cp -f build/tinygo bin/go
```

Comment:
Would a symlink work? Always copying the file is perhaps a bit overkill.
Alternatively, perhaps we could only put the built binary in bin/go but naming it go (and not tinygo) is perhaps confusing.

Reply:
Not sure yet, I'll try later on today and find out.

Are we expecting TinyGo to be built on Windows?  Symlinks probably wouldn't work there, and the path separators would likely need revising too.

Reply:
At some point we'll want a Windows build, yes. But in that case, `tinygo` would also be called `tinygo.exe` so more modifications will be needed. The blocker so far has been a good CI environment that supports build caching (appveyor has too many limitations, Azure Pipelines will soon have build caching but doesn't support that right now).

Reply:
Using a symlink instead does work.  Will update this PR in a sec.